### ISSUE TITLE
landingpage.html - Missing alt tags in <img>

### ISSUE THREAD

**Issue by nickmeagher** [nickmeagher/PrimerPartnerships#2]
A lot of the images on the landing page are missing alt tags which is super important for SEO / Accessibility.

Make sure these are added in and explain the image to the best of your ability.
